Tests/StreamChatTests/Database/DTOs/ChannelDTO_Tests.swift

```diff
@@ -487,7 +487,7 @@ final class ChannelDTO_Tests: XCTestCase {
             // Truncate the channel to leave only 10 newest messages
             // We're dropping 9 messages to fullfil the predicate: createdAt >= channel.truncatedAt"
             let truncateDate = channelDTO.messages
-                .sorted(by: { $0.createdAt < $1.createdAt })
+                .sorted(by: { $0.createdAt.bridgeDate < $1.createdAt.bridgeDate })
```

Comment:
```suggestion
                .sorted(by: { $0.createdAt < $1.createdAt })
```

Reply:
NSDate is not comparable, so converting both to Date makes sense